apps/vor/pkg/postgres/postgres.go

```diff
@@ -185,7 +186,14 @@ type School struct {
 	Curriculum   Curriculum
 	Guardian     []Guardian
 }
-
+type Attendance struct {
+	Id        string `json:"id" pg:",type:uuid"`
+	StudentId string `pg:"type:uuid,on_delete:CASCADE"`
+	Student   Student
```

Comment:
what does this field do if there's already `StudentId` field?

Reply:
To get the student details

Reply:
if we dont have 'Student Student', we won't be able to access the student's name from attendance result right?

Reply:
yes, that student field would also set StudentId as a foreign key.